docs/exchange-operators/faq.mdx

```diff
@@ -146,7 +146,7 @@ mina-archive-blocks --precomputed --archive-uri <postgres uri> FILE.
 
 ### How do I query for the canonical block at a certain height from the archive node
 
-Use a recursive query. See [Query the database](/node-operators/archive-node#query-the-database) examples in the Archive Node docs.
+Use a recursive query. See [Query the database](/node-operators/archive-nodes/getting-started#query-the-database) examples in the Archive Node docs.
```

Comment:
@halsaphi the docs have other links to https://docs.minaprotocol.com/node-operators/archive-node (singular) so when your major-upgrade branch is merged into main, we'll need to carefully update all of those links (or you can change back to singular `/node-operators/archive-node`)